<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_client">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_join">
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>client_join</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clientJoin">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clientGoMain">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>go back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_clientJoinAddress">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_chat">
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>client_chat</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_clientChatView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>600</width>
       <height>350</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clientExit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>0</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>go back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clientSend">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>420</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_clientWrite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>417</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_copy">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
